scripts/mongo/1347-attachment-version-off-by-one-mapping-script.js

```diff
@@ -0,0 +1,65 @@
+// Directly related to Enceladus issue #1347
```

Comment:
Could we add that this is 1.2.X to 2.5 problem?

Reply:
Done.